### PR TITLE
Fix azp artifacts download script pipeline ID

### PR DESCRIPTION
**Pull request by liushilongbuaa**
### Description of PR

Summary:
Fixes # (issue)
CI trigger artifacts does not contain BFN platform image
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### CODE REVIEW

File: tests/scripts/getbuild.py [hunk 1]
```diff
@@ -93,10 +93,15 @@ def download_artifacts(url, content_type, platform, buildid):
             print("Download error", e)
             sys.exit(1)
 
-def find_latest_build_id(branch):
+def find_latest_build_id(platform,branch):
```
Comment (lguohan) [inline tests/scripts/getbuild.py:96]: space

File: tests/scripts/getbuild.py [hunk 2]
```diff
@@ -111,17 +116,17 @@ def main():
 
     parser = argparse.ArgumentParser(description='Download artifacts from sonic azure devops.')
     parser.add_argument('--buildid', metavar='buildid', type=int, help='build id')
-    parser.add_argument('--branch', metavar='branch', type=str, help='branch name')
+    parser.add_argument('--branch', metavar='branch', type=str, default='master', help='branch name')
     parser.add_argument('--platform', metavar='platform', type=str,
-            choices=['broadcom', 'mellanox', 'kvm'],
+            choices=['broadcom', 'mellanox', 'vs', 'barefoot', 'generic'],
```
Comment (lguohan) [inline tests/scripts/getbuild.py:121]: can you add armhf, centec as well?

File: tests/scripts/getbuild.py [hunk 3]
```diff
@@ -111,17 +116,17 @@ def main():
...
             help='platform to download')
     parser.add_argument('--content', metavar='content', type=str,
             choices=['all', 'image'], default='image',
             help='download content type [all|image(default)]')
     args = parser.parse_args()
 
     if args.buildid is None:
-        buildid = find_latest_build_id(args.branch)
+        buildid = find_latest_build_id(args.platform,args.branch)
```
Comment (lguohan) [inline tests/scripts/getbuild.py:129]: space

File: tests/scripts/getbuild.py [hunk 4]
```diff
@@ -111,17 +116,17 @@ def main():
 
     parser = argparse.ArgumentParser(description='Download artifacts from sonic azure devops.')
     parser.add_argument('--buildid', metavar='buildid', type=int, help='build id')
-    parser.add_argument('--branch', metavar='branch', type=str, help='branch name')
+    parser.add_argument('--branch', metavar='branch', type=str, default='master', help='branch name')
```
Comment (lguohan) [inline tests/scripts/getbuild.py:119]: can you also add download type option, for example the aboot versus onie image. for onie iamge, can downlaod the bin, for aboot, then download swi  image.

Comment (lguohan) [reply tests/scripts/getbuild.py:119]: also add type to download debug v.s. non debug image.

**Review by lguohan — APPROVED** [review]

File: tests/scripts/getbuild.py [hunk 5]
```diff
@@ -72,13 +72,27 @@ def get_download_url(buildid, artifact_name):
...
+        filename = "sonic-{}.raw".format(platform)
+
+        url = url.replace('zip', 'file')
+        url += "&subPath=%2Ftarget%2F{}".format(filename)
+    elif content_type == 'dbg':
+        filename = "sonic-{}-dbg.bin".format(platform)
+
         url = url.replace('zip', 'file')
         url += "&subPath=%2Ftarget%2F{}".format(filename)
     else:
```
Comment (qiluo-msft) [inline tests/scripts/getbuild.py:98]: >else [](http://example.com/codeflow?start=4&length=4)

Do we truly has `else` cases?

File: tests/scripts/getbuild.py [hunk 6]
```diff
@@ -72,13 +72,27 @@ def get_download_url(buildid, artifact_name):
...
         else:
             filename = "sonic-{}.bin".format(platform)
 
+        url = url.replace('zip', 'file')
+        url += "&subPath=%2Ftarget%2F{}".format(filename)
+    elif content_type == 'aboot':
+        filename = "sonic-aboot-{}.swi".format(platform)
+
+        url = url.replace('zip', 'file')
+        url += "&subPath=%2Ftarget%2F{}".format(filename)
```
Comment (qiluo-msft) [inline tests/scripts/getbuild.py:87]: Is it possible to move both lines out of if-elif-else block?

**Comment by liushilongbuaa** [comment]
didn't need this script any more

**Pull request closed without merging**